arviz/plots/posteriorplot.py

```diff
@@ -24,6 +24,8 @@ def plot_posterior(
     group="posterior",
     rope=None,
     ref_val=None,
+    rope_color=None,
+    ref_val_color=None,
```

Comment:
I would set the default to `C1` and `C2` respectively, so you can use `vectorized_to_hex` directly down the line. You'll see that there are other string values as defaults. What should not be set as defaults are mutable objects such as dicts or lists which is why you'll not see a dict used as default, always none and the if none initialize the empty dict.

This is a common python gotcha, here is some more info in case you wanted to learn more about that: https://docs.python-guide.org/writing/gotchas/